### PR TITLE
Remove liquidity for nft burn

### PR DESCRIPTION
**Pull request by bogdan-rosianu**
in case of a ESDTNFTBurn operation, decrease the liquidity from system account

### CODE REVIEW

**Review by sasurobert — APPROVED** [review]

**Review by raduchis — APPROVED** [review]

**Review by iulianpascalau — DISMISSED** [review]

File: builtInFunctions/esdtDataStorage.go [hunk 1]
```diff
@@ -363,7 +363,6 @@ func (e *esdtDataStorage) saveESDTMetaDataToSystemAccount(
 	if len(currentSaveData) == 0 && e.flagSendAlwaysEnableEpoch.IsSet() {
 		esdtDataOnSystemAcc.Properties = nil
 		esdtDataOnSystemAcc.Reserved = []byte{1}
-		esdtDataOnSystemAcc.Value.Set(esdtData.Value)
```
Comment (iulianpascalau) [inline builtInFunctions/esdtDataStorage.go:366]: should not be breaking do to the use of flagSendAlwaysEnableEpoch flag :+1:

**Review by iulianpascalau — APPROVED** [review]

File: builtInFunctions/esdtDataStorage.go (hunk 1 above)
Comment (raduchis) [inline builtInFunctions/esdtDataStorage.go:366]: this line was added in https://github.com/ElrondNetwork/elrond-vm-common/commit/7346e85eca473e7161a6d3f3de028bb8ebe2d163 with an esdtStorageHandler.AddToLiquiditySystemAcc, similar to the one added in this PR. 
is it ok to be removed?

Comment (sasurobert) [reply builtInFunctions/esdtDataStorage.go:366]: Yes. It is ok

Comment (raduchis) [reply builtInFunctions/esdtDataStorage.go:366]: ok!

**Review by raduchis — APPROVED** [review]

**Pull request merged**